crates/orchestrator/src/jobs/types.rs

```diff
@@ -138,6 +140,56 @@ pub struct JobItem {
     pub updated_at: DateTime<Utc>,
 }
 
+/// Defining a structure that contains the changes to be made in the job object,
+/// id and created at are not allowed to be changed
+// version and updated_at will always be updated when this object updates the job
+#[derive(Serialize, Debug)]
+pub struct JobItemUpdates {
+    pub internal_id: Option<String>,
+    pub job_type: Option<JobType>,
+    pub status: Option<JobStatus>,
+    pub external_id: Option<ExternalId>,
+    pub metadata: Option<HashMap<String, String>>,
+}
+
+/// implements default selections for JobItemUpdates
+impl Default for JobItemUpdates {
+    fn default() -> Self {
```

Comment:
instead of default, let's rename this to `new`

Reply:
Alright, changes [here](https://github.com/madara-alliance/madara-orchestrator/pull/134/commits/881b3aac10f025ffe5e81fa15f80703e57c667dc).